src/browser-driver-manager.js

```diff
@@ -1,259 +1,124 @@
-          );
-        }
-      }
-    }
-
-    return;
-  }
-  // output version installed from npm
-  else if (['version', 'which'].includes(userArgs[0]) && chromedriver) {
-    try {
-      const chromedriverPkg = require('chromedriver');
+  return installedBrowser;
+}
 
-      if (userArgs[0] === 'version') {
-        console.log(chromedriverPkg.version);
-      } else {
-        console.log(chromedriverPkg.path);
-      }
-
-      return;
-    } catch (err) {
-      return error(chalk, 'chromedriver is not installed');
-    }
-  }
+async function setEnv({ chromePath, chromedriverPath }) {
+  console.log('Setting env CHROME/CHROMEDRIVER_TEST_PATH');
 
   try {
-    await runBashScript({
-      args: userArgs
-    });
-  } catch (err) {
-    return;
+    await fsPromises.writeFile(
+      path.resolve(BDM_CACHE_DIR, '.env'),
+      `CHROME_TEST_PATH="${chromePath}"\nCHROMEDRIVER_TEST_PATH="${chromedriverPath}"`
+    );
+    console.log(
+      'CHROME_TEST_PATH is set in ',
+      chromePath,
+      '\nCHROMEDRIVER_TEST_PATH is set in ',
+      chromedriverPath
+    );
+  } catch (e) {
+    console.error('Error setting CHROME/CHROMEDRIVER_TEST_PATH', e);
   }
 }
 
-/**
- * Output an error message
- * @param {Object} chalk
- * @param {String} msg - Message to display
- */
-async function error(chalk, msg) {
-  console.error(chalk.red('browser-driver-manager error:'), msg);
-  process.exitCode = 1;
-}
-
-/**
- * Output a message for verbose logging
- * @param {Object} chalk
- * @param {Boolean} verbose - If verbose logging is enabled
- * @param {String} msg - Message to display
- */
-async function verboseLog(chalk, verbose, msg) {
-  if (verbose) {
-    console.log(chalk.blue('log:'), msg);
+function getEnv() {
+  const envPath = path.resolve(BDM_CACHE_DIR, '.env');
+  if (!fs.existsSync(envPath)) {
+    return null;
   }
+  const env = fs.readFileSync(envPath, 'utf8');
+  return env;
 }
 
-/**
- * Uppercase first character of a string
- * @param {String} str - String to Titlecase
- */
-function titleCase(str) {
-  return str.charAt(0).toUpperCase() + str.substr(1);
-}
+async function browserDriverManager(args) {
+  if (args[0]?.trim() === 'which') {
+    const env = getEnv();
+    console.log(env);
+    return;
+  }
 
-/**
- * Run the main bash script
- * @param {Object} options
- * @param {Boolean} [options.returnValue] - Return the bash script output rather than display it
- * @param {Boolean} [options.sudo] - Run the command as sudo
- * @param {String[]} options.args - List of script arguments
- * @returns {Promise<String|null>}
- */
-function runBashScript({ returnValue = false, sudo = false, args } = {}) {
-  return new Promise((resolve, reject) => {
-    const lines = [];
-    let lastChunk;
+  if (args[0]?.trim() === 'version') {
+    const pattern = /-(\d+\.\d+\.\d+\.\d+)/;
+    const filePath = getEnv();
+    // Search for the pattern in the file path
+    const match = filePath?.match(pattern);
```

Comment:
`getEnv` does not return a path to a file but the contents of the env file, so this seems like a misnomer.

```suggestion
    const env = getEnv();
    // Search for the pattern in the env file
    const match = env?.match(pattern);
```